packages/platform-sdk/src/contracts/coins/ledger.ts

```diff
@@ -21,4 +23,6 @@ export interface LedgerService {
 	signTransaction(path: string, payload: Buffer): Promise<string>;
 
 	signMessage(path: string, payload: Buffer): Promise<string>;
+
+	scan(): Promise<WalletData[]>;
```

Comment:
Just want to be sure this is all we need, or whether we need any more params? Like start scanning from wallet index **n**, etc. Is it ok to return `WalletData[]`? It will be very basic data (like balance), not all data in `WalletData`.

Reply:
For now we shouldn't need any parameters but we'll see what we need once we add more coins and have to handle all their quirks.